src/Checkbox/Input.js

```diff
@@ -20,10 +20,7 @@ export class Input extends Component {
             value,
             onFocus,
             onBlur,
-            autocomplete,
```

Comment:
React does not like it when we use DOM attributes on React elements. I didn't expect this, but they aren't used anywhere so let's just backtrack and remove them.